distribution/kpkginstall/runtest.sh

```diff
@@ -160,34 +162,35 @@ function get_kpkg_ver()
   fi
 
   # Write the KVER to a file in /tmp so we have it after reboot.
-  echo -n "${KVER}" | tee -a /tmp/kpkginstall/KPKG_KVER
+  echo -n "${KVER}" > /tmp/kpkginstall/KPKG_KVER
 }
 
 function targz_install()
 {
   declare -r kpkg=${KPKG_URL##*/}
-  echo "Fetching kpkg from ${KPKG_URL}"
+  print_info "Fetching kpkg from ${KPKG_URL}"
 
-  if curl -OL "${KPKG_URL}" 2>&1; then
-    echo "✅ Downloaded kernel package successfully from ${KPKG_URL}"
+  if curl -sOL "${KPKG_URL}" 2>&1; then
+    print_success "Downloaded kernel package successfully from ${KPKG_URL}"
   else
     abort_recipe "Failed to download package from ${KPKG_URL}" WARN
   fi
 
-  echo "Extracting kernel version from ${KPKG_URL}"
+  print_info "Extracting kernel version from ${KPKG_URL}"
   get_kpkg_ver
   if [ -z "${KVER}" ]; then
     abort_recipe "Failed to extract kernel version from the package" FAIL
   else
-    echo "Kernel version is ${KVER}"
+    print_success "Kernel version is ${KVER}"
   fi
 
   if tar xfh ${kpkg} -C / 2>&1; then
-    echo "✅ Extracted kernel package successfully: ${kpkg}"
+    print_success "✅ Extracted kernel package successfully: ${kpkg}"
```

Comment:
The first emoji should be dropped from here, yeah?

Reply:
Good catch!